_posts/2022-05-31-scala-on-app-service.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Scala on App Service"
+toc: true
+toc_sticky: true
+author_name: Denver Brittain
+excerpt: "Run a Scala App using Java SE on App Service"
+category: java
+---
+

```

Comment:
Can you add an introduction here? Introduce concepts (Scala, Play framework) and then use the generic, _follow the tutorial below to deploy a Play framework Scala app onto Azure App Service._ 

Really helps with SEO ;)